api/src/main/java/org/openmrs/module/appointments/model/Appointment.java

```diff
@@ -197,5 +197,9 @@ public Boolean isFutureAppointment() {
         Date startOfDay = DateUtil.getStartOfDay();
         return this.getStartDateTime().after(startOfDay) || startOfDay.equals(this.getStartDateTime());
     }
+
+    public boolean isSameAppointment(Appointment appointment) {
```

Comment:
Overriding `equals` method better suits here?

Reply:
it may affect in some other places. Hence haven't changed.